src/RPCServer.ts

```diff
@@ -37,6 +37,37 @@ import * as events from './events';
 
 const cleanupReason = Symbol('CleanupReason');
 
+function composeMessage(error: unknown): string {
```

Comment:
The function name is too vague here. Something like `composeErrorMessage` would be better.